### PR TITLE
Make HDCP level configurable for RPI Platform

### PR DESCRIPTION
**Pull request by sramani-metro**

### CODE REVIEW

File: DisplayInfo/RPI/PlatformImplementation.cpp [hunk 1]
```diff
@@ -223,9 +241,38 @@ class DisplayInfoImplementation : public Exchange::IHDRProperties, public Exchan
...
+
+            static bool GetValue(const Core::JSON::Boolean& jsonValue) {
+                return jsonValue.IsSet() ? jsonValue.Value() : false;
+            }
+
+            static int32_t GetValue(const Core::JSON::DecUInt32& jsonValue) {
+                return jsonValue.IsSet() ? jsonValue.Value() : 0;
+            }
+
+            Core::JSON::String hdcpLevel;
```
Comment (MFransen69) [inline DisplayInfo/RPI/PlatformImplementation.cpp:273]: Couldn't we better use Core::JSON::EnumType<> ?

File: DisplayInfo/RPI/PlatformImplementation.cpp [hunk 2]
```diff
@@ -350,6 +397,7 @@ class DisplayInfoImplementation : public Exchange::IHDRProperties, public Exchan
     mutable Core::CriticalSection _adminLock;
 
     Core::WorkerPool::JobType< DisplayInfoImplementation& > _activity;
+    Config _config;
```
Comment (MFransen69) [inline DisplayInfo/RPI/PlatformImplementation.cpp:400]: Is there actually a reason to store the config for now? We already store the parse result in _value correct?

**Review by MFransen69 — CHANGES_REQUESTED** [review]

File: DisplayInfo/RPI/PlatformImplementation.cpp (hunk 1 above)
Comment (sramani-metro) [reply DisplayInfo/RPI/PlatformImplementation.cpp:273]: I thought having a string in the config file would help understand what it means. But yes, We could.

File: DisplayInfo/RPI/PlatformImplementation.cpp (hunk 2 above)
Comment (sramani-metro) [reply DisplayInfo/RPI/PlatformImplementation.cpp:400]: You are right. We dont need to store it at this time. I will change this

File: DisplayInfo/RPI/PlatformImplementation.cpp (hunk 1 above)
Comment (MFransen69) [reply DisplayInfo/RPI/PlatformImplementation.cpp:273]: The result for the config file itself would be the same. It is the implementation here that is easier to do (e.g. with support for the conversions etc)

Comment (sramani-metro) [reply DisplayInfo/RPI/PlatformImplementation.cpp]: Yes, it is better to use existing enum conversions. It does simplify the implementation by a great deal. Pushed new commit.

**Review by MFransen69 — APPROVED** [review]

**Pull request merged**